daemon/cmd/daemon.go

```diff
@@ -686,6 +696,26 @@ func NewDaemon(ctx context.Context, cancel context.CancelFunc, epMgr *endpointma
 	if len(restoredCIDRidentities) > 0 {
 		d.ipcache.UpsertGeneratedIdentities(restoredCIDRidentities)
 	}
+	// Upsert restored local Ingress IPs
+	nIngressIPs := 0
+	var firstErr error
+	for _, ingressIP := range oldIngressIPs {
+		_, err := d.ipcache.Upsert(ingressIP.String(), nil, 0, nil, ipcache.Identity{
+			ID:     identity.ReservedIdentityIngress,
+			Source: source.Local,
+		})
+		if err == nil {
+			nIngressIPs++
+		} else if firstErr == nil {
+			firstErr = err
+		}
+	}
+	if firstErr != nil {
+		log.WithError(firstErr).WithField(logfields.Count, fmt.Sprintf("%d", len(oldIngressIPs)-nIngressIPs)).Error("error while restoring ingress identities")
```

Comment:
@tommyp1ckles That was a remnant from a version that also restored IngressIPs of other nodes, wanted to avoid flooding the logs with likely similar errors in case there are a lot of nodes. Now that we restore only local Ingress IPs we can log all errors.

Reply:
Changed to a warning level.